src/index.html

```diff
@@ -13,10 +13,16 @@
     <title>Stop watch</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/blocks/index.scss"
```

Comment:
The stylesheet is linked as an SCSS file (`index.scss`). Typically, browsers do not directly interpret SCSS files. Ensure that the SCSS is compiled to CSS and the link is updated to point to the resulting CSS file.